doc/wg/network/notes/network-notes-2023-08-24.md

```diff
@@ -0,0 +1,102 @@
+# Tock Network WG Meeting Notes 2023-08-10
+
+- **Date:** August 24th, 2023
+- **Participants:**
+    - Alex Radovici
+    - Tyler Potyondy
+    - Branden Ghena
+    - Leon Schuermann
+    - Felix Mada
+    - Amit Levy
+- **Agenda**
+    1. Updates
+    2. Alexandru: Solution for drawing during meetings
+    3. Tyler: Interfaces for different layers used for Thread networking
+    4. Leon: Tock-ethernet integration strategy
+- **References:**
+    - [NRF52840 802.15.4 PR](https://github.com/tock/tock/pull/3615)
+    - [STM32F429 Ethernet PR](https://github.com/tock/tock/pull/3523)
+
+## Updates
+- Branden: There's a label for Network WG related PRs on the Tock github repo
+- Tyler: Working on 15.4 PR. Need some closure on behavior for transmit
+- Branden: We had a discussion about handling on/off states for the 15.4 radio. Other things in Tock like GPIO just "do the right thing" and set themselves in the state they need to be. The 15.4 radio HIL expects callbacks when changing power states though, which likely makes sense for external radio chips and radios that take some real-world time to start up.
+- [TODO Fill in notes from recording]
```

Comment:
TODO

Reply:
Reminder to handle these TODOs @lschuermann

Reply:
Yup sorry! I had some more time critical things come up, but I'll try to have this done today.

Reply:
No rush! Just wanted to remind you it exists.